### ISSUE TITLE
Dockerimage for the agent version 2 - ports

### ISSUE THREAD

**Issue by timea** [Praqma/noops#33]
Based on our last meeting

- open the port on the actual docker machine to the external network so the agent can be called

I need to research this, implement and make a small writeup so users using different docker machines can set it up with ease.

**Comment by buep** [comment]
Yep, the dockerhost need to accept trafic in on the ports... be aware that only one agent can then run on the host, unless you chose another port the the second docker container.

Any, keep this issue on hold for a short while, because after our droplets was compromised we don't want to expose also the docker host port. Then everyone can run containers on our host.
